tasks/install.yml

```diff
@@ -5,12 +5,72 @@
     state: installed
   with_items:
     - java-1.8.0-openjdk
-    - maven
     - ant
     - tomcat
     - tomcat-webapps
     - tomcat-admin-webapps
 
+- name: add JAVA_OPTS to tomcat.conf
```

Comment:
My recommendation would be to leave tomcat.conf as a jinja template, rather than using lineinfile. Right now, we just want to change one variable, but it's likely we'll want to make other changes in the future. Also, as written, this will replace the original line, but after that I think it will start adding repeated declarations of the JAVA_OPTS variable when we re-run the role.